### DISCUSSION TITLE
Brand suggestions

### DISCUSSION THREAD

**Discussion by asongtoruin** [jpanther/lynx#5] [Ideas]
Hey James! Congrats on a lovely theme. I was having a look through the brand icons on Font Awesome, and have the following suggested icons and colours:

![image](https://user-images.githubusercontent.com/20771863/143130297-cb6c299b-0505-42b4-b93c-b31d89a1825e.png)

There's two options for Goodreads on Font Awesome - I think the second might be better?

Let me know what you think - I can pull together a PR (though I'm not quite sure all the places the changes would need to be added!)

**Comment by jpanther** [comment]
Hey @asongtoruin, thanks for the suggestions. I'm happy to include more brands and these ones all look like useful additions. I agree that the second Goodreads looks cleaner. If you're happy to submit a PR I'll take a look, otherwise I can look to add these into a future update.

To answer your question, the only places that need to be updated are the icon SVG file, the CSS with the button style and the string in the i18n file.

**Comment by pesader** [comment]
Hi! I've made a PR (#52) to include some of the brands you suggested, namely Stack Overflow and Stack Exchange.
